suites/pacific/upgrades/tier-1_upgrade_cephadm.yaml

```diff
@@ -226,3 +226,12 @@ tests:
           ceph_origin: rhcs
           fsid: f64f341c-655d-11eb-8778-fa163e914bcc
           ireallymeanit: 'yes'
+  - test:
```

Comment:
Hey Mohit,

The tc aims to check whether purge cluster (playbook) leaves any cluster logs. Since this suite already had a test case to validate the purge steps, it would be ideal to validate the log deletion as part of it. Please let me know your thoughts.

Reply:
Thanks @pranavprakash20 
Can you please check once with "Test Owner" regarding the test suite.

Reply:
IMO, we can add this test case in this suite. 

The PR looks good to me.

Reply:
Thanks "Pranav"
LGTM